File: launchdarkly/csharp/src/OSEG.LaunchDarklyExamples/CreateOAuth2ClientExample.cs
using System;
using System.Collections.Generic;
using System.IO;
using System.Text.Json;

using Org.LaunchDarklyTools.Api;
using Org.LaunchDarklyTools.Client;
using Org.LaunchDarklyTools.Model;

namespace OSEG.LaunchDarklyExamples;

public class CreateOAuth2ClientExample
{
    public static void Run()
    {
        var config = new Configuration();
        config.ApiKey.Add("Authorization", "YOUR_API_KEY");

        var oauthClientPost = new OauthClientPost(
        );

        try
        {
            var response = new OAuth2ClientsApi(config).CreateOAuth2Client(
                oauthClientPost: oauthClientPost
            );

            Console.WriteLine(response);
        }
        catch (ApiException e)
        {
            Console.WriteLine("Exception when calling OAuth2ClientsApi#CreateOAuth2Client: " + e.Message);
            Console.WriteLine("Status Code: " + e.ErrorCode);
            Console.WriteLine(e.StackTrace);
        }
    }
}

File: launchdarkly/typescript-axios/src/CreateOAuth2ClientExample.ts
import axios, { AxiosError } from "axios";
import * as api from "launchdarkly_client"

const configuration = new api.Configuration({
  apiKey: "YOUR_API_KEY",
});

const oauthClientPost: api.OauthClientPost = {
};

new api.OAuth2ClientsApi(configuration).createOAuth2Client(
  oauthClientPost,
).then(response => {
  console.log(response.data);
}).catch((error: Error | AxiosError) => {
  console.log("Exception when calling OAuth2ClientsApi#createOAuth2Client:");
  axios.isAxiosError(error) ? console.log(error.message) : console.log(error);
});
